package.json

```diff
@@ -52,7 +53,7 @@
     "watchify": "^3.7.0"
   },
   "dependencies": {
-    "popsicle": "^8.2.0",
+    "any-promise": "^1.3.0",
```

Comment:
Sorry, I just absolutely hate configuring every module I want to work with to use `Promise`. If anything, this could just be upgraded to assume `Promise` already exists and say `>= 4` from now on.